### PR TITLE
Formatting cleanup to increase readability

### PR DESCRIPTION
**Pull request by sytone**

### CODE REVIEW

File: source/_components/ecobee.markdown [hunk 1]
```diff
@@ -15,9 +15,32 @@ featured: true
...
+1. Click on the **Become a developer** link on the [developer site](https://www.ecobee.com/developers/). 
+2. Login with your Ecobee credentials.
```
Comment (Landrash) [inline source/_components/ecobee.markdown]: Mix of Ecobee & ecobee in documentation.

File: source/_components/ecobee.markdown [hunk 2]
```diff
@@ -15,9 +15,32 @@ featured: true
...
+3. Give your app a name (it appears to need to be unique across all users, as I tried 'home-assistant' and it said it was already in use. Try <yournameoralias>-home-assistant) and a summary (neither of these are important as they are not used anywhere). 
+4. For Authorization method select **ecobee PIN**. 
+5. You don't need an Application Icon or Detailed Description. 
+6. Click **Save**. 
+
+Now under the Name and Summary Section you will have an API key. Copy this key and use it in you configuration section below. Click the **X** to close the Developer section. 
+
+The first time you run Home Assistant with this component it will give you a PIN code that you need to authorize in the [ecobee consumer portal](https://www.ecobee.com/consumerportal/index.html). You can do this by clicking **Add Application** in the **My Apps** section in the sidebar. 
+
+The PIN can be found from the Home Assistant portal on the Ecobee card or from the **configurator.ecobee** entity in states in the portal.
```
Comment (Landrash) [inline source/_components/ecobee.markdown]: Mix of Ecobee & ecobee in documentation

File: source/_components/ecobee.markdown (hunk 1 above)
Comment (fabaff) [inline source/_components/ecobee.markdown]: The company called themself "ecobee".

**Comment by fabaff** [comment]
Can you please also update the configuration variables, please? According to @nkgilley the API key is only needed for the first setup.

Thanks.

**Comment by sytone** [comment]
Updated the documentation for comments.

**Comment by fabaff** [comment]
Thanks

**Pull request merged**